src/deploy/osp_deployer/profiles/xsp.json

```diff
@@ -274,6 +282,14 @@
                   "false"
                ]
             },
+=======
+        }
+
+       }
+      },
+      {
+         "PowerFlex Settings": {
+>>>>>>> 7f7dd247a955e87fefb48b6c56ee23923c4f2675
```

Comment:
Needs to be resolved

Reply:
Don't know why it's there. Supposed to got it resolved.